c7n/resources/iam.py

```diff
@@ -1119,6 +1119,91 @@ def process(self, resources, event=None):
         return []
 
 
+# ############### begin  CIEM Role filter ##################
+
+@Role.filter_registry.register('has-inline-policy-attached')
+class IamRoleInlinePolicy(Filter):
+    """
+        Filter IAM roles that have an inline-policy attached
+
+        Y: Filter roles that have an inline-policy and attached the policy document
+        N: Filter roles that do not have an inline-policy
+        Both: will not filter but attached inline policy document if any user have it.
+
+    """
+
+    schema = type_schema('has-inline-policy-attached', value={'type': 'string'})
+    permissions = ('iam:ListRolePolicies', 'iam:GetRolePolicy')
+
+    def _role_inline_policies(self, client, resource, value):
+        inlinePolicies = client.list_role_policies(RoleName=resource['RoleName'])['PolicyNames']
+        resource['c7n:InlinePolicies'] = []
+        for inlinePolicy in inlinePolicies:
+            inlinePolicyDocument = client.get_role_policy(RoleName=resource['RoleName'], PolicyName=inlinePolicy)
```

Comment:
Please fix the variable name:

https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function